controller/src/main/java/org/dswarm/controller/resources/resource/ResourcesResource.java

```diff
@@ -1051,7 +1050,7 @@ private ProxyConfiguration addConfiguration(final Resource resource, final Strin
 	 * @return the new persisted configuration
 	 * @throws DMPControllerException
 	 */
-	private ProxyConfiguration createNewConfiguration(final ConfigurationService configurationService) throws DMPControllerException {
+	private static ProxyConfiguration createNewConfiguration(final ConfigurationService configurationService) throws DMPControllerException {
```

Comment:
In general, this is micro-optimization thing. static methods exists only once, where instance methods exist for every live instance. When a method doesn't use fields the IDE suggest to replace it with a static method. This is probably an instance where such an auto-correct from the IDE slipped through.


> On Mar 27, 2015, at 9:59 AM, Bo Ferri <notifications@github.com> wrote:
> 
> In controller/src/main/java/org/dswarm/controller/resources/resource/ResourcesResource.java:
> 
> > @@ -1051,7 +1050,7 @@ private ProxyConfiguration addConfiguration(final Resource resource, final Strin
> >  	 * @return the new persisted configuration
> >  	 * @throws DMPControllerException
> >  	 */
> > -	private ProxyConfiguration createNewConfiguration(final ConfigurationService configurationService) throws DMPControllerException {
> > +	private static ProxyConfiguration createNewConfiguration(final ConfigurationService configurationService) throws DMPControllerException {
> y static here?
> 
> —
> Reply to this email directly or view it on GitHub.
>